README.md

```diff
@@ -1,141 +1,146 @@
+
+The staking and un-staking actions are governed by the following formulas:
+
+**Staking:**
+
+![](http://latex.codecogs.com/gif.latex?s&space;=&space;t&space;*&space;\(\(1&space;&plus;&space;d&space;/&space;b\)^r-1)
+
+**Unstaking:**
+
+![](http://latex.codecogs.com/gif.latex?c&space;=&space;b&space;*&space;\(1&plus;s&space;/&space;t\)^{\frac{1}{r}})
+
+Where `s` is the resulting stake in the court and `d` is the users deposit. When un-staking, `c` is the amount of ANT returned. The other variables represent state and parameters of the staking curve. `r` is the ratio of deposits to total stake, `b` is the balance of ANT in the staking contract, and `t` is the total amount of distributed stake.  
+
+Once a prospective juror has staked they are considered *active* and eligible to be drafted to review cases. A distinction is made between a juror being *staked* and *active.* An *active* juror can *de-activate* themselves at any time, but they will not be able to *un-stake* until any pending disputes they have been involved in have completed.
 
-# 1. Introduction
-The Bitcoin network has proven to be a disruptive force on the global financial system. Bitcoin's innovation is broad but the most significant contributions can be summarized as (1) combining cryptography and economics to create a novel toolkit for solving large scale distributed coordination challenges and (2) using that toolkit to secure transactions on a global peer-to-peer cash system independent of any centralized authority.
+### Juror Drafting
 
-Building on the innovations of Bitcoin, Ethereum created a platform for decentralized applications that abstracts away the crypto-economic security of block production from application developers. By separating concerns, application developers can share a common security layer and thereby pool the cost of securing the network. Pooling the cost of security reduces the portion of value that individual applications must allocate to security compared to running an independent app-chain.
+Juror drafting is managed via a process of *stake-weighted sortition*. In order to manage the sortition process efficiently, all of the court's operations are scheduled into *Terms*. *Terms* are defined in seconds and cannot be changed after the Court has been initialized. Terms are transitioned by calling a public *heartbeat* function which is used to make updates to *active juror stakes* from the preceding term and generate a new *random seed* for use during the subsequent term. A portion of the courts fees are used to compensate the caller of the heartbeat for gas usage.
```

Comment:
Might we want to use a word other than "Terms" here, so as not to cause confusion with the same word used in the context of "terms of a contract"?

Reply:
@izqui @bingen I think this is a good point, but also want weigh the impact this has on the clarity of the actual implementation.

Reply:
I don't have a strong opinion on this. I picked the name after reading [this](https://en.wikipedia.org/wiki/Legal_year). I guess we could also call it periods, but it seems more generic and less precise.

Reply:
The phrase totally makes sense in this context, it's just that since we are also using "terms" to refer to the content of a proposal agreement, I thought it might be worth trying to avoid a terminology clash. Perhaps as long as we make the context of usage clear when using these words there will be no confusion. Just something to keep in mind :)

Reply:
I totally understand the possible confusion but I don't have strong opinions about it. Poor knowledge of legal jargon (even in my native language) plus non-native English, haha :wink: